epam-promo/components/tables/ColumnHeaderDropdown/SortingPanel.tsx

```diff
@@ -13,22 +13,21 @@ export interface SortingPanelProps {
 }
 
 const SortingPanelImpl: React.FC<SortingPanelProps> = ({ sortDirection, onSort }) => {
-    const sortAsc = useCallback(() => onSort('asc'), [onSort]);
-    const sortDesc = useCallback(() => onSort('desc'), [onSort]);
+    const sortAscDesc = useCallback((dir: SortDirection) => onSort(dir === sortDirection ? undefined : dir), [onSort]);
 
     return (
         <FlexCell cx={ css.sortingPanelContainer }>
             <DropdownMenuButton
                 isActive={ sortDirection === 'asc' }
                 caption={ i18n.pickerFilterHeader.sortAscending }
                 icon={ SortIcon }
-                onClick={ sortAsc }
+                onClick={ () => sortAscDesc('asc') }
```

Comment:
you wrap sortAscDesc into useCallback, but here use arrow function, so useCallback is useless